src/components/trends/smbg/FocusedSMBGPointLabel.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+import Tooltip from '../common/Tooltip';
+
+import { MGDL_UNITS, MMOLL_UNITS } from '../../../utils/constants';
+import { displayBgValue } from '../../../utils/format';
+import { formatDisplayDate, millisecondsAsTimeOfDay } from '../../../utils/datetime';
+
+import styles from './FocusedSMBGPointLabel.css';
+
+// tooltip offsets
+const SIMPLE_VALUE_TOP_OFFSET = 15;
+const SIMPLE_DAY_TOP_OFFSET = 10;
+const SIMPLE_DAY_LEFT_OFFSET = 30;
+const DETAILED_DAY_LEFT_OFFSET = 5;
+
+const FocusedSMBGPointLabel = (props) => {
+  const { focusedPoint } = props;
+  if (!focusedPoint) {
+    return null;
+  }
+
+  const {
+    bgUnits,
+    focusedPoint: { data, position, dayPoints, positions },
+    timePrefs,
+    lines,
+  } = props;
+
+  let parsedTime;
+  if (timePrefs.timezoneAware) {
+    if (data.time) {
+      parsedTime = Date.parse(data.time);
+    }
+  } else {
+    if (data.deviceTime) {
+      parsedTime = Date.parse(data.deviceTime);
+    }
+  }
+  const smbgTime = formatDisplayDate(parsedTime, timePrefs, 'dddd MMM D');
+  const shortDate = formatDisplayDate(parsedTime, timePrefs, 'MMM D');
+  const pointTooltips = _.map(dayPoints, (smbg, i) => (
+    <Tooltip
+      key={i}
+      content={<span className={styles.number}>{displayBgValue(smbg.value, bgUnits)}</span>}
+      position={positions[i]}
+      side={'bottom'}
+      tail={false}
+      offset={{ top: SIMPLE_VALUE_TOP_OFFSET, left: 0 }}
+    />
+  ));
+  let focusedTooltip;
+  if (lines) {
+    focusedTooltip = (<Tooltip
+      title={<span className={styles.explainerText}>{smbgTime}</span>}
+      position={position}
+      side={'right'}
+      offset={{ top: SIMPLE_DAY_TOP_OFFSET, left: SIMPLE_DAY_LEFT_OFFSET }}
+    />);
+  } else {
+    focusedTooltip = (<Tooltip
+      title={<span className={styles.tipWrapper}>
```

Comment:
Just a note that as discussed we should be mindful of the potential need to abstract out a TooltipTitle and/or TooltipContent component(s) to avoid duplicating the `tipWrapper` styles in many places.